### DISCUSSION TITLE
phishing web site

### DISCUSSION THREAD

**Discussion by cyberturkeyyy** [community/community#136284] [Copilot Conversations]
### Select Topic Area

General

### Body

Hello,

The reported URL address redirects to a phishing website. We kindly request your valuable assistance in closing the address.

`Sensitive info removed by staff
`

PLEASE NOTE THAT: THIS IS URL WORKING ONLY WITH ENTER TURKEY IP'S (TURKEY LOCATIONS USERS) IF YOU WILL ENTER WITH TURKEY LOCATION VPN SERVICE YOU WILL SEE PHISHING CONTENT

Kind Regards

**Comment by pierrelouisguichard** [comment]
Thanks for the warning

**Reply by cyberturkeyyy** [reply]
Accounts need to be closed urgently, can you please help on the issue?

Thank you!

**Comment by Akash1134** [comment]
Hi @cyberturkeyyy , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by cyberturkeyyy** [comment]
Hello,

We keep reporting but the accounts are closed. I constantly create a support request, but the accounts are not closed, can you help me with this issue?

Thank you

Kimden: Akash ***@***.***>
Tarih: Pazartesi, 19 Ağustos 2024 18:15
Kime: community/community ***@***.***>
Bilgi: cyberturkeyyy ***@***.***>, Mention ***@***.***>
Konu: Re: [community/community] phishing web site (Discussion #136284)

Hi @cyberturkeyyy<https://github.com/cyberturkeyyy> , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools<https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam>. Here's all the info:

  *   Reporting abuse or spam<https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam>
  *   Reporting a user<https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user>
  *   Reporting an issue or pull request<https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request>
  *   Reporting a comment<https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment>

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
